.travis.yml

```diff
@@ -30,7 +32,7 @@ script:
 
 # Build App in Debug and Run Tests if specified
   - if [ $RUN_TESTS == "YES" ]; then
-      travis_retry xcodebuild -workspace "$WORKSPACE" -scheme "$SCHEME" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO ENABLE_TESTABILITY=YES test;
+      travis_retry xcodebuild -workspace "$WORKSPACE" -scheme "$SCHEME" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO ENABLE_TESTABILITY=YES APP_ID="$APP_ID" APP_CODE="$APP_CODE" test | xcpretty;
```

Comment:
BTW, if the trailing `;` is not required here (and in line 37), should we just take the chance to get rid of it?